lib/info.js

```diff
@@ -108,10 +111,32 @@ module.exports = function getInfo(link, options, callback) {
     } else {
       dashmpd2 = results[1].dashmpd;
     }
+    config.args.relatedIds = relatedIds;
     gotConfig(options, description, config, dashmpd2, callback);
   });
 };
 
+/**
+ * @param {String} HTML body
+ */
+function getRelatedVideos(body) {
+  // Get the RELATED_PLAYER_ARGS from body then parse for the video Ids
+  var related = body.match(/RELATED_PLAYER_ARGS['\s\t\n]*:[\s\t\n]*{[\s\t\n"]*rvs":"([^}]+)/);
```

Comment:
Still think this regex can be simplified. And since the format of the "rvs" string seems consistent, we could probably use something like `related[1].split(',').map(function(s) { return querystring.parse(s); })` which would also retrieve a bit for data of the related videos.